pkgs/tools/package-management/nix/default.nix

```diff
@@ -195,12 +195,12 @@ in rec {
 
   nixFlakes = lib.lowPrio (callPackage common rec {
     name = "nix-2.3${suffix}";
-    suffix = "pre20190712_aa82f8b";
+    suffix = "pre20190829_ebc4dae";
     src = fetchFromGitHub {
       owner = "NixOS";
       repo = "nix";
-      rev = "aa82f8b2d2a2c42f0d713e8404b668cef1a4b108";
-      hash = "sha256-MRY2CCjnTPSWIv0/aguZcg5U+DA+ODLKl9vjB/qXFpU=";
+      rev = "ebc4dae51761cb54c1db28cf4d328d3f2f55b758";
+      hash = "08rx6k15rrcjjdzmwzifvvsr6vpz0n5cccy9hv441fz5pvl7kyn9";
```

Comment:
How is the hash calculated here?

Reply:
`hash` should be an SRI hash, which can be computed using `nix hash-file` or `nix to-sri`, e.g.
```
$ nix to-sri --type sha256 08rx6k15rrcjjdzmwzifvvsr6vpz0n5cccy9hv441fz5pvl7kyn9
sha256-yfp56L7lu0DIhskzxooF/26T9d4ufl5/k5LlXMI0PSM=
```